src/app/map/search/search.component.ts

```diff
@@ -24,24 +26,28 @@ export class SearchComponent implements OnInit, OnChanges, OnDestroy {
   @Input() projects: Array<Project> = [];
   @Input() selected: string;
   @Output() updateMatching = new EventEmitter();
+  @Output() showPlace = new EventEmitter();
 
   public loading = false;
   public ranSearch = false;
   public mineFilter: string = null;
   public _mineFilter: string = null; // temporary filters for Cancel feature
   public permitFilter: string = null;
-  public _permitFilter: string = null; // temporary filters for Cancel feaure
+  public _permitFilter: string = null; // temporary filters for Cancel feaure     is this necessary?
```

Comment:
seems like the _***Filter variables are all not really needed. In the end they just get copied over to the non-underscored version anyway in the `applyFilters` function and elsewhere?

`this.permitFilter = this._permitFilter;`

It probably makes sense at some point to just use a single var. Not really super important to change it all now though, but yeah, probably not necessary!